ansible/playbook/files/supervisord.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Process Monitoring and Control Daemon
+After=rc-local.service nss-user-lookup.target
+After=network.target
+RequiresMountsFor=/srv
+
+[Service]
+Type=simple
+# we need to wait till time is synchronized
+ExecStartPre=/usr/bin/sleep 10
+ExecStart=/usr/bin/supervisord -n -c /etc/supervisord.conf
```

Comment:
Do we manage supervisord.conf somewhere? How it is generated? Does it contain `nodaemon`? Should we add `nodaemon` as a flag there to ensure supervisord does not try to daemonize itself?

Reply:
so I added `-n` which is the same as `--nodaemon`.
We manage supervisord.conf in percona-images, I don't think that it requires changes there.
https://github.com/Percona-Lab/percona-images/blob/1e731407d4b98fd35bf059bc4edd0baa918b6ae6/ansible/roles/supervisord-init/tasks/main.yml#L6

If you mean supervisord.service, we also have it there, but I'm not sure that it will update supervisord.service file on updating pmm.
https://github.com/Percona-Lab/percona-images/blob/master/ansible/roles/supervisord-init/files/supervisord.service

Reply:
Doh! Missed that. Thanks!